playbooks/frontend.yml

```diff
@@ -23,3 +24,12 @@
         - perl-XML-LibXML
         - unzip
         - curl
+
+    - stat: path={{ scripts_dir }}
+      register: scripts_dir_check
+
+    - name: Setup Amazon CloudWatch Memory Monitoring Script
+      import_tasks: '../tasks/cloudwatch_memory_monitoring.yml'
+      vars:
+        user_scripts_dir: "{{ scripts_dir }}"
+      when: scripts_dir_check.stat.exists == False
```

Comment:
Would it not make more sense to be checking the `CloudWatchMonitoringScripts-1.2.2` directory as well?  In the case that the scripts dir is created by something else?